conformity/fields/email.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import, unicode_literals
+
+import re
+
+import attr
+
+from conformity.error import Error
+from conformity.fields.basic import UnicodeString
+from conformity.utils import strip_none
+from conformity.fields.net import (
+    IPAddress,
+)
+
+
+@attr.s
+class EmailAddress(UnicodeString):
+    """
+    On the shoulder of mighty Django
+    https://github.com/django/django/blob/master/django/core/validators.py#L164
+    """
+    message = 'Enter a valid email address.'
+    code = 'invalid'
+    ip_schema = IPAddress()
+
+    user_regex = re.compile(
+        r"(^[-!#$%&'*+/=?^_`{}|~0-9A-Z]+(\.[-!#$%&'*+/=?^_`{}|~0-9A-Z]+)*\Z"  # dot-atom
+        r'|^"([\001-\010\013\014\016-\037!#-\[\]-\177]|\\[\001-\011\013\014\016-\177])*"\Z)',  # quoted-string
+        re.IGNORECASE)
```

Comment:
It would be nice to match the code style of the rest of the codebase and put the trailing `)` on the next line, but I'm not going to block on that :)

Reply:
Format edited